core/validation/validation.go

```diff
@@ -61,7 +61,6 @@ func IsWorkload(spiffeid string) bool {
 					" val: " + env.SpiffeIdPrefixForWorkload() +
 					" trust: " + env.SpiffeTrustDomain(),
 			)
-			return false
 		}
```

Comment:
Let's keep these `return`s. 

Yes, you cannot return after a `panic`; yet, it is still useful to maintain signature parity.

Reply:
OK I'll add back few minutes later 👍